### PR TITLE
Update translation of documentation/installation (ja)

### PR DESCRIPTION
**Pull request by maimux2x**
## やったこと
https://www.ruby-lang.org/en/documentation/installation/#package-management-systems
上記英語版Rubyインストールページのパッケージ管理システムの箇所に合わせて、日本語版ドキュメントの説明が足りない箇所を修正。

### CODE REVIEW

File: ja/documentation/installation/index.md [hunk 1]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
 {% endhighlight %}
 
+こちらはRubyの最新の安定版をインストールすることを推奨します。
 
 ### Homebrew (macOS)
 {: #homebrew}
 
-Ruby 2.0.0 は OS X El Capitan、Yosemite および Mavericks に含まれています。
-また、OS X Mountain Lion、Lion および Snow Leopard には 1.8.7 が含まれています。
+Ruby2.0以降はEI Capian(10.11)のリリース以降よりMacOSに始めから含まれています。
```
Comment (sorah) [inline ja/documentation/installation/index.md:193]: 誤訳

```suggestion
El Capitan (10.11) のリリース以降、macOS には Ruby 2.0 以上が同梱されています。
```

File: ja/documentation/installation/index.md [hunk 2]
```diff
@@ -148,6 +144,22 @@ $ sudo yum install ruby
 
 インストールされるバージョンは、一般に、特定のディストリビューションのバージョンがリリースされた時点での、Ruby の最新バージョンです。
 
+### snap(Ubuntu または他のLinux distribution)
```
Comment (sorah) [inline ja/documentation/installation/index.md:147]:
```suggestion
### snap (Ubuntu やその他の Linux distribution)
```

File: ja/documentation/installation/index.md [hunk 3]
```diff
@@ -148,6 +144,22 @@ $ sudo yum install ruby
...
+{: #snap}
+
+SnapはCanonialによって開発されたパッケージ管理システムです。こちらはUbuntu上で利用が可能な点に加えて他の多数のLinuc distributionでも動かすことができます。
```
Comment (sorah) [inline ja/documentation/installation/index.md:150]:
```suggestion
SnapはCanonialによって開発されたパッケージ管理システムです。こちらはUbuntu上で利用が可能な点に加えて他の多数のLinux distributionでも動かすことができます。
```

File: ja/documentation/installation/index.md [hunk 4]
```diff
@@ -148,6 +144,22 @@ $ sudo yum install ruby
...
+以下のように実行します。
```
Comment (sorah) [inline ja/documentation/installation/index.md:151]:
```suggestion
以下のように利用できます:
```

File: ja/documentation/installation/index.md [hunk 5]
```diff
@@ -148,6 +144,22 @@ $ sudo yum install ruby
...
+
+{% highlight sh %}
+$ sudo snap install ruby --classic
+{% endhighlight %}
+
+Rubyにはいくつかのマイナーシリーズがあります。例えば以下のように実行するとRuby2.3に切り替えることが可能です。
```
Comment (sorah) [inline ja/documentation/installation/index.md:157]: 誤訳

```suggestion
マイナーシリーズごとの channel が用意されています。例えば以下を実行するとRuby 2.3に切り替えることが可能です。
```

File: ja/documentation/installation/index.md [hunk 6]
```diff
@@ -148,6 +144,22 @@ $ sudo yum install ruby
...
+
+{% highlight sh %}
+$ sudo snap switch ruby --channnell=2.3/stable
```
Comment (sorah) [inline ja/documentation/installation/index.md:160]:
```suggestion
$ sudo snap switch ruby --channnel=2.3/stable
```

File: ja/documentation/installation/index.md [hunk 7]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
+こちらはRubyの最新の安定版をインストールすることを推奨します。
 
 ### Homebrew (macOS)
 {: #homebrew}
 
-Ruby 2.0.0 は OS X El Capitan、Yosemite および Mavericks に含まれています。
-また、OS X Mountain Lion、Lion および Snow Leopard には 1.8.7 が含まれています。
+Ruby2.0以降はEI Capian(10.11)のリリース以降よりMacOSに始めから含まれています。
+[Homebrew](https://brew.sh/)はmacOSで利用されているパッケージ管理システムです。
+Homebrewを使うことでRubyを簡単にインストールできます。
```
Comment (sorah) [inline ja/documentation/installation/index.md:195]:
```suggestion
[Homebrew](https://brew.sh/) は macOS で広く利用されているパッケージ管理システムです。Homebrew で Ruby をインストールするのは下記のように簡単です:
```

File: ja/documentation/installation/index.md [hunk 8]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
 
-すでに 2.0 も 1.8 も古いバージョンです。
-そのため、Ruby の最新バージョンをインストールするためのいくつかの方法があります。
+{% highlight sh %}
+$ brew install ruby
+{% endhighlight %}
 
-Ruby コミュニティにいる大半の macOS ユーザは Ruby をインストールするためにサードパーティ製のツールを使用しています。
-しかし、いくつかのパッケージマネージャが Ruby をサポートしています。
+こちらはRubyの最新の安定版をインストールすることを推奨します。
```
Comment (sorah) [inline ja/documentation/installation/index.md:202]: 誤訳

```suggestion
上記コマンドによって最新の安定版がインストールされます。
```

File: ja/documentation/installation/index.md [hunk 9]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
 
-多くの macOS ユーザはパッケージマネージャとして [Homebrew][homebrew] を使っています。
-これを使うと本当に簡単に Ruby を手に入れることができます:
+### FreeBSD
+{: #pkg}
+FreeBSDでは、Rubyをインストールする方法として、パッケージ化されたものとソースベースのものの両方が用意されています。プリビルドパッケージはpkgツールでインストールできます。
```
Comment (sorah) [inline ja/documentation/installation/index.md:206]: https://docs.freebsd.org/ja/books/handbook/ports/#pkgng-intro に合わせて訳語を選択

```suggestion
FreeBSD においては Ruby のインストールに、バイナリパッケージと ports が利用可能です。バイナリ配布パッケージは pkg でインストールできます:
```

File: ja/documentation/installation/index.md [hunk 10]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
-$ brew install ruby
+$ pkg install ruby
 {% endhighlight %}
 
-また、 macOS は Unix ベースなので、ソースコードをダウンロードしてインストールするのも、
-他の方法と同じように簡単で効果的な方法です。
-macOS 上で新しい Ruby のバージョンをインストールする手助けのために、
-サードパーティ製ツールを使うことも良い方法だと考えられます。
+[Ports Collection](https://docs.freebsd.org/en/books/handbook/ports/#ports-using)を使用してRubyをインストールする場合、ソースベースの方法を使用することが可能です。
+こちらはビルドの設定ツールをカスタマイズしたい場合に便利です。
```
Comment (sorah) [inline ja/documentation/installation/index.md:213]: https://docs.freebsd.org/ja/books/handbook/ports/#pkgng-intro に合わせて訳語を選択、あと誤訳

```suggestion
ports を利用してソースコードからインストールする場合、[Ports Collection](https://docs.freebsd.org/en/books/handbook/ports/#ports-using) を利用してください。この方法はビルド設定をカスタマイズする場合に便利です。
```

File: ja/documentation/installation/index.md [hunk 11]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
 
+FreeBSDにおけるRubyと関連するエコシステムのもっと詳しい情報は[FreeBSD Ruby Projects website](https://wiki.freebsd.org/Ruby) で確認が可能です。
```
Comment (sorah) [inline ja/documentation/installation/index.md:215]:
```suggestion
FreeBSDにおけるRubyとそのエコシステムの詳しい情報は [FreeBSD Ruby Projects website](https://wiki.freebsd.org/Ruby) で確認できます。
```

File: ja/documentation/installation/index.md [hunk 12]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
+
+{% highlight sh %}
+$ doas pkg_add ruby
+{% endhighlight %}
 
-[Sunfreeware][sunfreeware] で Solaris 8 から 10 用の Ruby 1.8.7 が使用できます。
-Blastwave で Ruby 1.8.7 が使用できます。
-[Sunfreeware][sunfreeware] で Ruby 1.9.2p0 も使用できますが、これは古いバージョンです。
-サードパーティ製ツールを使用することで最新バージョンの Ruby を手に入れることができます。
+複数のメジャーバージョンを並列でインストールすることができます。なぜかというと、それらのバイナリは異なる名前を持っているからです (例: ruby27, ruby26)。
```
Comment (sorah) [inline ja/documentation/installation/index.md:225]:
```suggestion
複数のメジャーバージョンを共存させインストールすることができます。それらのバイナリは異なる名前を持っているからです (例: ruby27, ruby26)。
```

File: ja/documentation/installation/index.md [hunk 13]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
 
-[OpenIndiana][openindiana] で Ruby をインストールするには、
-[Image Packaging System (IPS)][opensolaris-pkg] クライアントを使ってください。
-これは Ruby バイナリと RubyGems を直接 OpenSolaris ネットワークリポジトリからインストールします:
+`OpenBSD` のportsコレクションの `HEAD` ブランチには、このプラットフォーム用のRubyの最新版がリリースされてから一定期間経ったものが含まれている可能性があり、[directory lang/ruby in the most recent ports collection](https://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/lang/ruby/?only_with_tag=HEAD)を確認してください。
```
Comment (sorah) [inline ja/documentation/installation/index.md:227]: 誤訳

```suggestion
リリース間も無い最新版の Ruby は OpenBSD の ports collection の `HEAD` ブランチで提供されている場合があります。 [最新 ports collections の lang/ruby](https://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/lang/ruby/?only_with_tag=HEAD) もあわせて確認してください。
```

File: ja/documentation/installation/index.md [hunk 14]
```diff
@@ -173,57 +185,80 @@ Ruby を手に入れるには、次のようにしてください:
...
+
+### Ruby on OpenIndiana
+{: #openindiana}
+[OpenIndiana](https://www.openindiana.org/)にRubyをインストールするには、ImagePackagingSystem(IPS)クライアントを利用してください。これは、RubyバイナリとRubyGemsをOpenIndianaのリポジトリから直接インストールするものです。簡単です。
```
Comment (sorah) [inline ja/documentation/installation/index.md:231]:
```suggestion
[OpenIndiana](https://www.openindiana.org/) に Ruby をインストールするには、Image Packaging System (IPS) クライアントを利用してください。これは、 Ruby バイナリと RubyGems  をOpenIndiana のリポジトリから直接インストールするものです。簡単です:
```

**Review by sorah — COMMENTED** [review]
Thank you for contribution. I found some mistranslations in your submission, please confirm and apply my suggestions to move this forward.

**Comment by maimux2x** [comment]
@sorah 
Thank you for your detailed review. I confirmed your suggestions and applied. 
I am afraid that there are many points that I did not check properly.
As I check more carefully, I will continue to work on correcting the installation page of other section.

**Pull request merged**